detd/manager.py

```diff
@@ -94,9 +98,18 @@ def add_talker(self, config):
                 self.talker_manager[config.interface.name] = interface_manager
 
             return self.talker_manager[config.interface.name].add_talker(config)
+    
+    def add_listener(self, config):
 
+        logger.info("Adding listener to Manager")
 
+        with self.lock:
+
+            if not config.interface.name in self.listener_manager:
+                interface_manager = InterfaceManager(config)
```

Comment:
As a post-merge task, we should have a single InterfaceManager instance for both talker and listener streams. No need to change in this PR.